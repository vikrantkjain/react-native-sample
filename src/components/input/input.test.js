import {render, fireEvent, screen} from '@testing-library/react-native';
import Input from './index';

describe('Test Input', () => {
  const onChangeTestMock = jest.fn();
  const changeText = 'Hello';
  const tree = render(<Input onChangeText={onChangeTestMock} />);
  it('Render successfully', () => {
    expect(tree).toBeDefined();
  });
  it('Render Correctly', () => {
    const treeToJSON = tree.toJSON();
    expect(treeToJSON).toMatchSnapshot();
  });
  fireEvent.changeText(screen.getByTestId('TEXTINPUT-1'), changeText);
});
