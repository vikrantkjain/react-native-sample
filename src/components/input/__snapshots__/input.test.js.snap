// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Input Render Correctly 1`] = `null`;
